### PR TITLE
Fix plugin navigation

### PR DESCRIPTION
**Pull request by cogor**
Described in Issue #7730

### CODE REVIEW

File: shell/components/Tabbed/index.vue [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default {
 
     useHash: {
       type:    Boolean,
-      default: true,
+      default: false,
```
Comment (cnotv) [inline shell/components/Tabbed/index.vue:35]: You have to update all the use cases of Tabbed component if you want to revert this key. 
The alternative is that you set it true for the case you want to apply it.

**Review by cnotv — CHANGES_REQUESTED** [review]
Mind the E2E outcome.

**Review by richard-cox — CHANGES_REQUESTED** [review]
There should be a way to fix this without updating all references.

To help us with that we need to reproduce the bug (see comments in https://github.com/rancher/dashboard/issues/7730)

**Comment by richard-cox** [comment]
Closing as per conversation in https://github.com/rancher/dashboard/issues/7730

**Pull request closed without merging**